extensions/authentication-jwt/README.md

```diff
@@ -164,6 +166,46 @@ The code snippet for whoAmI function:
   }
 ```
 
+### End points with refresh token
+
+To add refresh token mechanism in your app. you have to call the following
+interrceptor.
+
+1. ('refresh-token-generate') : to generate the refresh token and access token
```

Comment:
Maybe using this style instead to be consistent with the rest of the documentation?

```suggestion
1. `refresh-token-generate` : to generate the refresh token and access token
```